authors/apps/profiles/models.py

```diff
@@ -0,0 +1,34 @@
+from django.db import models
+from authors.settings import AUTH_USER_MODEL
+from authors.apps.core.models import TimestampMixin
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from authors.apps.authentication.models import User
+from cloudinary.models import CloudinaryField
+
+
+class UserProfile(TimestampMixin, models.Model):
+    """
+    This model create user profile once a user account is created.
+    The user profile comprises of bio and the image field
+    """
+    user = models.OneToOneField(AUTH_USER_MODEL, on_delete=models.CASCADE,
+                                related_name='profile')
+    bio = models.TextField(default="Update your profile")
+    image = CloudinaryField("image", default="https://res.cloudinary.com/dy2faavdk/image/upload/v1548264034/qvxtpdmi03kksg9rxgfj.png")
+
+    @property
+    def username(self):
+        return self.user.username
+
+    def __str__(self):
+        return self.user.username
+
+    @receiver(post_save, sender=User)
+    def create_user_profile(sender, instance, created, **kwargs):
+        if created:
```

Comment:
Write a description as to what create_user_profile function does and what save_user_profile does

Reply:
@Collins33  I have added the description to create_user_profile and save_user_profile methods